pkg/apis/cassandraoperator/v1alpha1/zz_generated.openapi.go

```diff
@@ -210,10 +210,12 @@ func schema_pkg_apis_cassandraoperator_v1alpha1_CassandraDataCenterSpec(ref comm
 				Properties: map[string]spec.Schema{
 					"cluster": {
 						SchemaProps: spec.SchemaProps{
-							Ref: ref("k8s.io/api/core/v1.LocalObjectReference"),
+							Description: "Cluster is either a string or v1.LocalObjectReference Cluster interface{} `json:\"cluster,omitempty\"`",
+							Type:        []string{"string"},
+							Format:      "",
 						},
 					},
-					"replicas": {
+					"nodes": {
```

Comment:
Will we need to keep renaming this / will changing the autogen stuff break things down the line?